packages/tocco-ui/src/SearchBox/SearchBox.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import Button from '../Button'
+import './styles.scss'
+import _debounce from 'lodash/debounce'
+
+/**
+ * SearchBox
+ */
+class SearchBox extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      inputValue: ''
+    }
+    this.search = _debounce(this.search, props.debounce)
+  }
+
+  updateValue = evt => {
+    this.setState({'inputValue': evt.target.value})
+  }
+
+  search = () => {
+    const value = this.state.inputValue
+    if (this.props.liveSearch || (value && value.length >= this.props.minInputLength)) {
+      this.props.onSearch(value)
+    }
+  }
+
+  keyDownHandler = evt => {
+    const KEY_ENTER_CODE = 13
+    if (this.props.liveSearch || evt.keyCode === KEY_ENTER_CODE) {
+      this.search()
+    }
+  }
+
+  render() {
+    return (
+      <div
+        className="tocco-searchbox row">
+        <div
+          className="col-lg-6">
```

Comment:
sure we want to set this class? i think a generic component should not be limited in this way